core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -1,100 +1,92 @@
-<ng-container *ngIf="editorState$ | async as es">
-    <dot-edit-ema-toolbar (editUrlContentMap)="editContentMap($event)" />
-    <dot-results-seo-tool
-        *ngIf="es.editorData.mode === editorMode.SOCIAL_MEDIA && ogTags() && ogTagsResults$"
-        [seoMedia]="es.editorData.socialMedia"
-        [seoOGTags]="ogTags()"
-        [seoOGTagsResults]="ogTagsResults$"
-        data-testId="results-seo-tool"></dot-results-seo-tool>
+@if(uveStore.editorState(); as es) {
```

Comment:
We might needs to be able to remove this, no?

Reply:
No, because the state is being lazy loaded, so we would have errors